tensorflow_addons/optimizers/rectified_adam.py

```diff
@@ -78,7 +78,7 @@ def __init__(
         weight_decay: Union[FloatTensorLike, Callable, Dict] = 0.0,
         amsgrad: bool = False,
         sma_threshold: FloatTensorLike = 5.0,
-        total_steps: int = 0,
+        total_steps: FloatTensorLike = 0.0,
```

Comment:
How about `TensorLike = 0`?

Reply:
If `_get_hyper` with dytpe doesn't work It could be a good compromise to relax the type hint